tests/observability/metrics/utils.py

```diff
@@ -1283,6 +1285,99 @@ def get_metric_labels_non_empty_value(prometheus: Prometheus, metric_name: str)
     return {}
 
 
+def get_pod_memory_stats(admin_client: DynamicClient, hco_namespace: str, pod_prefix: str) -> int:
+    return int(
+        bitmath.Byte(
+            int(
+                get_pod_by_name_prefix(
+                    dyn_client=admin_client,
+                    pod_prefix=pod_prefix,
+                    namespace=hco_namespace,
+                ).execute(command=RSS_MEMORY_COMMAND)
+            )
+        ).MiB
+    )
+
+
+def get_highest_memory_usage_virt_api_pod_dict(hco_namespace: str) -> dict[str, Any]:
```

Comment:
returns pod + memory usage, tuple here will make it easier to consume the returned values (i.e no need to access any keys).
a dict in this case adds no extra value, as only one pod + mem wll be returned